src/regional_settings.cpp

```diff
@@ -264,6 +264,19 @@ static void load_overmap_feature_flag_settings( const JsonObject &jo,
                 overmap_feature_flag_settings.whitelist.emplace( line );
             }
         }
+
+        if( overmap_feature_flag_settings_jo.has_array( "special_counts" ) ) {
+            JsonArray specials = overmap_feature_flag_settings_jo.get_array( "special_counts" );
+
+            for( const JsonValue member : specials ) {
```

Comment:
Does not look right - `member` is not used.

Should probably be something like this?

```C++
            for( const JsonObject &o : specials ) {
                std::string flag = o.get_string( "flag" );
                int count = o.get_int( "count" );

                overmap_feature_flag_settings.special_counts.insert( std::make_pair( flag, count ) );
            }
```